docs/general-terminology/open-source.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: Open Source
+edit: https://github.com/pbclife/gitopener.vercel.app/edit/main/docs/open-source.mdx
+---
+
+# Open Source
```

Comment:
I think What is Open Source? would be most prominent as its a question.

Reply:
Yeah changing now..

Reply:
I am really sorry actually I forgot that time the changes will work on guides directory. Will you move `general terminology` folder inside `docs/guides` directory.

Reply:
I made changes but it added to my forked repo may i know the steps to follow next

Reply:
Just move your created folder and file inside `guides` --> `/docs/guides` instead of just `/docs`

Reply:
How to? Please guide this part...

Reply:
- create a directory named `general-terminology` inside `docs/guides` directory
- inside `general-terminology` create a file `open-source.mdx`
- add a title in the front-matter section
- in the body add a heading followed by your contribution

Reply:
Done the changes but if i do the Pull request the code is storing in my fork repo is it correct?
How to add it to main one?

Reply:
Yeah 👍🏾

Reply:
Done please check